### PR TITLE
Closes #1 simple event component

### LINKED ISSUE #1: Simple Event Component
Child of #2

Acceptance Criteria:
The Event Component displays:
 - event name
 - event start time
 - event end time

### PR DESCRIPTION
**Pull request by Aristarkh4**

### CODE REVIEW

File: packages/client/src/pages/Events/index.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ import Header from '../../components/Layout/Header'
 import Main from '../../components/Layout/Main'
 
 const event = {
-  title: 'Doing the mahi',
-  location: 'Lyall Bay'
+  name: 'WJ4J Library Hangout',
+  start_time: '13:00',
+  end_time: '15:00'
```
Comment (reubenberghan) [inline packages/client/src/pages/Events/index.js]: The properties `start_time` and `end_time` should probably be in camel case

File: packages/client/src/components/Event/index.js [hunk 2]
```diff
@@ -1,5 +1,41 @@
...
+    color: #fff;
+    margin: 0;
+    padding: 10px;
+    position: absolute;
+    text-align: center;
+    width: 180px;
+  }
+`
+
+export default function Event ({ name, start_time, end_time }) {
```
Comment (reubenberghan) [inline packages/client/src/components/Event/index.js]: The props here should also be in camel case

Comment (Aristarkh4) [reply packages/client/src/components/Event/index.js]: Fixed it with the latest commit.

File: packages/client/src/pages/Events/index.js (hunk 1 above)
Comment (Aristarkh4) [reply packages/client/src/pages/Events/index.js]: Fixed it with the latest commit.

File: packages/client/src/components/Event/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import Event from '.'
+
+describe('components:Event', () => {
+  it('should render the Event component CSS properly', () => {
+    expect(
+      toJson(
+        shallow(
+          <Event
+            name={'WJ4J Library Hangout'}
```
[the remark this answers is not in the document]
Comment (chasm) [reply packages/client/src/components/Event/index.spec.js]: Hello! Any action on this? Can we get this fixed so we can merge it?

[the remark this answers is not in the document]
Comment (nickmangos) [reply packages/client/src/components/Event/index.spec.js]: Commit cfbac83

File: packages/client/src/components/Event/index.js [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+  return (
+    <EventWrapper>
+      <h2>{name}</h2>
```
[the remark this answers is not in the document]
Comment (nickmangos) [reply packages/client/src/components/Event/index.js]: Commit cfbac83

**Comment by Aristarkh4** [comment]
Yes, it does. Can close without merging. Then the pull request for #15 will close both.

**Pull request closed without merging**